### PR TITLE
MVP-with-beautiful-interfaces

### PR DESCRIPTION
**Pull request by seafruit**

### CODE REVIEW

File: entity/dish.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by yoyo on 16-8-13.
+ */
+/**
+ * Created by SONY on 2016/8/11.
+ */
```
Comment (baixue1016) [inline entity/dish.js:3]: 注释的都删掉

File: entity/order.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by yoyo on 16-8-13.
+ */
+/**
+ * Created by SONY on 2016/8/11.
+ */
```
Comment (baixue1016) [inline entity/order.js:3]: 注释删掉

File: entity/dish.js (hunk 1 above)
Comment (baixue1016) [inline entity/dish.js:3]: 把var都改成let

File: mongodb/execute.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+};
+exports.finsDishInfoById = function (req, res) {
+    let id = req.body.id;
+    Dish.findOne({_id: id}, function (err, dishes) {
+        if (err) throw err;
+        res.send(dishes);
+    })
+};
+exports.saveOrder = function (req, res) {
+    // console.log('saveOrder++++++++++++++++\n'+ req.body);
```
Comment (baixue1016) [inline mongodb/execute.js]: 注释删掉

File: public/mealinfo.jsx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+          alert('预定成功！')
+        } else {
+          alert('预定失败,服务器忙,请稍后重试。')
+        }
+      })
+    }
+  }
+}
+
+// return (
```
Comment (baixue1016) [inline public/mealinfo.jsx]: 注释删掉

**Comment by jydawn** [comment]
:+1:

**Comment by goddessting** [comment]
:+1:

**Pull request merged**